### DISCUSSION TITLE
Migrating from Azure.Mobile.Server and Azure.Mobile.Client

### DISCUSSION THREAD

**Discussion by JPhilC** [Azure/azure-mobile-apps#506] [Q&A] [answered]
Around 4 years ago I wrote a WPF desktop client and azure app service using the Microsoft.Azure.Mobile.Client and Microsoft.Azure.Mobile.Server nugets. Am I correct in thinking that azure-mobile-apps is the successor to those two nugets?

If so, is there any guidance anywhere outlining the requirements/work required for updating to use the modern nugets?

**Comment by adrianhall** [comment, marked as answer]
Yes - the libraries here are the successors.  There is no guidance other than the documentation.

For the server side, it's a complete re-write - ASP.NET Core is very different from ASP.NET Framework and they are not compatible at all.
For the client side, you should be focused on the authorization, as we have abstracted that.  However, the core mechanisms of offline sync are the same, so it should be an easier migration.

**Reply by JPhilC** [reply]
Thanks for the info. By the way, I found a long article of yours very useful when I started the project. Glad to see you are still involved.
